sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/AadLoginTest.java

```diff
@@ -15,8 +16,9 @@
 
     @BeforeClass
     public static void setup() {
+        String endpoint = String.format("https://%s.blob.core.windows.net", ConfigurationManager.getConfiguration().get("PRIMARY_STORAGE_ACCOUNT_KEY"));
```

Comment:
PRIMARY_STORAGE_ACCOUNT_KEY. It should be account name

Reply:
This class is being deleted in another PR